examples/tutorial/tutorial_notebooks/introduction/exercise_1.ipynb

```diff
@@ -115,9 +117,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "AssertionError",
+     "evalue": "Not quite correct. Remember the single line of code change (See above)",
```

Comment:
Did you mean to leave the output? It looks less clean, and it reveals the answer for the user. Better to have the user run the cell and get the AssertionError themself.

Reply:
@jeffreykennethli I deleted not needed outputs